src/installer/installer.go

```diff
@@ -183,6 +186,20 @@ func (i *installer) startBootstrap() error {
 		return err
 	}
 
+	var mc types.MachineConfig
+	yamlFile := "/opt/openshift/openshift/99_openshift-machineconfig_99-master-ssh.yaml"
+	fieldName := "Spec.Config.Passwd.Users[0].SshAuthorizedKeys"
+	newSshKey, err := ioutil.ReadFile("/etc/ssh/ssh_host_rsa_key.pub")
+	if err != nil {
+		i.log.Error(err)
+		return err
+	}
+	err = i.patchMastersIgnition(yamlFile, fieldName, string(newSshKey), "append-list", &mc)
```

Comment:
what will happen if the user didn specify ssh key at all - it's not a mandatory field

Reply:
Umm that require some changes, I need to check if that change the yaml structure or just have an empty list (in the second case no changes will be needed)

Reply:
I guess we can create our own manifest "99_openshift-0assisted-installer-ssh.yaml" and place it under /opt/openshift/openshift/

Reply:
This may be a problem, the manifest contain an `ignition version` for example, so if the version change (in `openshift/installer`) it will break.

Another option is to insert a dummy shh key in case the user didn't supplied one and to override it with the bootstrap key in `assisted-installer`.

Reply:
Updated PR.